src/v2/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -96,7 +96,7 @@ export class Accept extends Component<AcceptProps> {
     logger.error(error)
     switch (error.code) {
       case "capture_failed": {
-        const parsedData = get(error, e => JSON.parse(e.data), {})
+        const parsedData = get(error, (e) => JSON.parse(e.data), {})
```

Comment:
I was commenting on a MP PR of @sweir27 yesterday, is this a new Prettier/setting? Feels a bit verbose, so many paranthesis!

Reply:
I agree -- prettier v2's "breaking change" was setting parans to default. That said i can understand the decision, since its always kinda a headache  to go back and add adition args or destructure an object or something; at scale this saves a lot of time. 

Need to look at our prettier settings and get this stuff consistent / run over repo.